ZenPacks/zenoss/WBEM/datasources/WBEMDataSource.py

```diff
@@ -31,16 +31,15 @@
 from ZenPacks.zenoss.PythonCollector.datasources.PythonDataSource \
     import PythonDataSource, PythonDataSourcePlugin
 
-from ZenPacks.zenoss.WBEM.modeler.wbem import check_if_complete
+from ZenPacks.zenoss.WBEM.modeler.wbem_agent import check_if_complete
```

Comment:
I don't see a `ZenPacks.zenoss.WBEM.modeler.wbem_agent` module. So this will fail.